src/wireviz/wv_colors.py

```diff
@@ -19,10 +43,17 @@
     'TQ': '#00ffff',
     'BU': '#0066ff',
     'VT': '#8000ff',
-    'BN': '#666600',
+    'BN': '#a52a2a',
+    'SL': '#708090',
+    # Faux-copper look, for bare CU wire
+    'CU': '#d6775e:#895956',
+    # Silvery look for tinned bare wire
+    'SN': '#aaaaaa:#84878c',
+    # Yellow-green PE wire
+    'PE': '#54aa85:#f7f854:#54aa85',
```

Comment:
Do we need separate yellow and green tones for PE? I'd rather use the existing YE and GN

Reply:
I can change those to the same. To be honest I kinda just grabbed the yellow and green for it from a random standards document. I am going to make a separate issue to standardize the wire colors, so that others can comment if necessary.  I'm going to make the file-level changes to the repo, then start fixing colors.

Reply:
please remove PE since it's not a color. it should be implemented as GNYE / YEGN like any other banded/striped wire.